src/info/mod.rs

```diff
@@ -408,13 +406,31 @@ mod tests {
         config.input = repo.path().to_path_buf();
         let info = Info::new(&config)?;
         let info_str = format!("{}", info);
-        let expected_info = fs::read_to_string(repo.path().join("..").join("expected"))?;
-        let escaped_expected_info = regex::escape(&expected_info);
-        let expected_info_expression =
-            escaped_expected_info.replace("MATCH_UNTIL_END_OF_LINE", ".*");
-        let re = Regex::new(format!(r#"{}"#, expected_info_expression).as_str()).unwrap();
-        if !re.is_match(&info_str) {
-            assert_eq!(info_str, expected_info);
+        let info_u8 = strip_ansi_escapes::strip(&info_str)?;
+        let simple_info_str = std::str::from_utf8(&info_u8)?;
+        let expected_info = [
+            r"(?s)onefetch-committer-name ~ git version .+",
+            r"-{37,}",
+            r"Project: repo",
+            r"HEAD: .+ \(main\)",
+            r"Created: 22 years ago",
+            r"Language:",
+            r".+Verilog \(100.0 %\).+",
+            r"Author: 100% author 2",
+            r"Last change: 22 years ago",
+            r"Repo: https://github.com/user/repo.git",
+            r"Commits: 2",
+            r"Lines of code: 1",
+            r"Size: 6 B \(1 file\)",
+        ]
+        .join(".");
```

Comment:
I'd be interested in moving this to a separate file, for example `tests/regex/test_language_repo.stdout.regex` or something, and using `include_str!`. The regex could potentially be more accurate by checking explicitly for newlines (`\r?\n`) instead of using the `s` flag.
Also, [the `x` flag](https://docs.rs/regex/latest/regex/#example-replacement-with-named-capture-groups) may be worth looking into to help make the regex more readable.

Reply:
Learning a lot in that last post, thanks for pointing those methods out.